bzip2/tests/test.bats

```diff
@@ -0,0 +1,5 @@
+@test "Version matches plan" {
+  TEST_PKG_VERSION="$(echo "${TEST_PKG_IDENT}" | cut -d/ -f3)"
+  actual_version="$(hab pkg exec "${TEST_PKG_IDENT}" bzip2 --help 2>&1 | grep -Po "(?!=bzip2, a block-sorting file compressor\.  Version )(1\.0\.6)")"
```

Comment:
Fix the hardcoded version 1.0.6 in the test

Reply:
Fixed.

Reply:
Did you review your own pull request?